README.md

```diff
@@ -36,29 +49,38 @@ For an equality assertion failure, the error with context might look as follows:
 >   Actual: "barfood"
 > ```
 
-To achieve this, each exercise should define a `.meta/test_messages.json` file. This file is a mapping of the test method's name (the `"name"` value in the [test runner output][test-runner-output-format]) to the message displayed when the test fails:
+To achieve this, each exercise should define a `tests` key in its `.meta/config.json` file. This key is a mapping of the test method's name (the `"name"` value in the [test runner output][test-runner-output-format]) to the command that we tell the user was run when the test fails:
 
 ```json
 {
-  "MyTestMethod": {
-    "msg": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
-  }
+  "tests": [
+    {
+      "name": "MyTestMethod",
+      "cmd": "MyTests.MyTestMethod(1)"
+    }
+  ]
 }
 ```
 
-Note that you can embed the [test runner's][test-runner-output-format] output (usually the assertion error message) in the message by using `%{output}`.
-
-As most test messages will be equal except for the called method, there is a shorthand notation that only requires the variable part (the method/command being called) to be specified:
+If you would like to provide a custom message (rather than the default interpolated message that we provide), you can also add a "msg" key, which must contain the command, and should in most cases also embed the [test runner's][test-runner-output-format] output (usually the assertion error message) in the message by using `%{output}`. For example:
 
 ```json
 {
-  "MyTestMethod": {
-    "cmd": "MyTests.MyTestMethod(1)"
-  }
+  "tests": [
+    {
+      "name": "MyTestMethod",
+      "cmd": "MyTests.MyTestMethod(1)",
+      "cmd": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
```

Comment:
```suggestion
      "msg": "We tried running `MyTests.MyTestMethod(1)` but got the following error:\n\n%{output}"
```

We could consider also having a `%{cmd}` placeholder available, as "also add a "msg" key" seems to imply that the `cmd` key is required.

```suggestion
      "msg": "We tried running `%{cmd}` but got the following error:\n\n%{output}"
```